sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -477,10 +476,20 @@ def send_message(
         visibility_timeout = kwargs.pop('visibility_timeout', None)
         time_to_live = kwargs.pop('time_to_live', None)
         timeout = kwargs.pop('timeout', None)
-        self._config.message_encode_policy.configure(
-            require_encryption=self.require_encryption,
-            key_encryption_key=self.key_encryption_key,
-            resolver=self.key_resolver_function)
+        try:
+            self._config.message_encode_policy.configure(
+                require_encryption=self.require_encryption,
+                key_encryption_key=self.key_encryption_key,
+                resolver=self.key_resolver_function,
+                encryption_version=self.encryption_version)
+        except TypeError:
+            warnings.warn(
+                "message_encode_policy.configure is likely missing the encryption_version parameter."
```

Comment:
I like this nuanced warning! Perhaps something more general at the start (just to let them know generally what happened, just in case `message_encode_policy.configure` is not their issue). 
Not 100% sure what to put here so open to your input, but something like "Encryption failed" or whatever happened, and then this warning would be nice.

Reply:
Yeah, that's a good suggestion. Will update.